docs/authentication/web3/metamask.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: MetaMask Wallet
```

Comment:
I believe it's better to use just `MetaMask` as that's how everyone refers to it